src/App.jsx

```diff
@@ -14,33 +16,94 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_FIELD_ALPHABET = 'ALPHABET';
+const SORT_FIELD_LENGTH = 'LENGTH';
 
```

Comment:
It's better to keep constants in a separate file. This way, you can easily manage all your constants in one place and avoid potential naming conflicts. Consider creating a new file to store constants like `SORT_FIELD_ALPHABET` and `SORT_FIELD_LENGTH`.